packages/safety-suite-api/src/api/userManagement/users.ts

```diff
@@ -0,0 +1,74 @@
+import {Request} from '@idelic/safety-net';
+
+import {runApi} from '../../runApi';
+import {ApiOptions, ApiResponse, CustomerSpecificQuery, Id} from '../../types';
+import {User} from './types';
+
+export interface GetUsersQuery extends Partial<CustomerSpecificQuery> {
+  /**
+   * Filter users by these ids.
+   */
+  ids?: Id[];
+}
+
+/**
+ * Get a list of users.
+ * @param query Object containing query params for this route.
+ * @param apiOptions Optional options for runApi.
+ * @returns Array of `User` objects.
+ */
+export const getUsers = (
+  query: GetUsersQuery = {},
+  apiOptions?: ApiOptions
+): Request<ApiResponse<User[]>> =>
+  runApi({
+    method: 'GET',
+    urlRoot: 'userManagementUrlRoot',
+    route: '/api/users',
+    apiOptions,
+    requestOptions: {
+      query
+    }
+  });
+
+/**
+ * Get a specific user by id.
+ * @param id Get user by this id.
+ * @param query Object containing query params for this route.
+ * @param apiOptions Optional options for runApi.
+ * @returns A single `User` object.
+ */
+export const getUser = (
+  id: Id,
+  query: CustomerSpecificQuery,
+  apiOptions?: ApiOptions
+): Request<ApiResponse<User>> =>
+  runApi({
+    method: 'GET',
+    urlRoot: 'userManagementUrlRoot',
+    route: `/api/users/${id}`,
```

Comment:
This is a Machine-to-Machine endpoint and it should be removed.
I will remove this endpoint from API Reference in scope of this PR: https://github.com/idelic-inc/api-reference/pull/34